utils/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -29,7 +29,8 @@ import com.here.ort.spdx.SpdxExpression
 object DeclaredLicenseProcessor {
     private val prefixesToRemove = listOf(
         "https://choosealicense.com/licenses/",
-        "https://opensource.org/licenses/"
+        "https://opensource.org/licenses/",
+        "https://www.gnu.org/licenses/old-licenses/"
```

Comment:
On 2nd thought I propose we remove "https://www.gnu.org/licenses/old-licenses/" here or change it to just "https://www.gnu.org/licenses"  else one looses in SPDXLicenseMapping the 'old-licenses' bit tells you that it's (L)GPL version 2 family and not 3.